packages/dependency-extraction-webpack-plugin/test/fixtures/bundled-packages/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * External dependencies
+ */
+import { isEmpty } from 'lodash';
```

Comment:
Note that those imports might need an ESLint ignore now that we're fully removing Lodash in #52571 

```suggestion
// eslint-disable-next-line no-restricted-imports
import { isEmpty } from 'lodash';
```

Reply:
Thanks for the heads-up. 

I changed `lodash` to `react-dom` in the test fixture. I think it's more sustainable and future-proof to test against something we plan to support in the longer term instead of silencing warnings.